app/webpack/stylesheets/application.scss

```diff
@@ -66,6 +66,8 @@
 @import "gto/venues.scss";
 @import "gto/components/index.scss";
 
+@import "openstudios/application.scss";
```

Comment:
also I know we're already mixing but maybe from here on we can use `open_studios` instead of `openstudios`